File: 2025-1/sesion-de-clase-04/PUCP/Edu/Pe/Inf30/Sesion04/Entidades/EntidadBase.cs
using System;

namespace PUCP.Edu.Pe.Inf30.Sesion04.Entidades {
	public abstract class EntidadBase {
		public int Codigo { get; set; }
		public string Nombre { get; set; }
		
		public virtual void imprimir() {
			Console.WriteLine("Codigo: " + this.Codigo);
			Console.WriteLine("Nombre: " + this.Nombre);
		}
	}
}

File: 2025-1/sesion-de-clase-04/pe/edu/pucp/inf30/sesion04/entidades/EntidadBase.java
package pe.edu.pucp.inf30.sesion04.entidades;

public abstract class EntidadBase {
	private int codigo;
	private String nombre;
	
	public int getCodigo() {
		return codigo;
	}
	
	public String getNombre() {
		return nombre;
	}
	
	public void setCodigo(int codigo) {
		this.codigo = codigo;
	}
	
	public void setNombre(String nombre) {
		this.nombre = nombre;
	}
	
	public void imprimir() {
		System.out.println("Codigo: " + this.codigo);
		System.out.println("Nombre: " + this.nombre);
	}
}